README.md

```diff
@@ -111,32 +108,27 @@ uv run pytest
 
 In order for some of the tests to run, they will need to hit actual APIs.  In order to do this,
 they will need to be properly configured with keys and URLs.  These need to be placed in the .env file.
+
 Note that if you have hard-coded values in the compose.yml files, they will override any hardcoded values in the .env file.
 
-For Google drive tests, update your .env with values shown below / pulled from vault as indicated:
+For Google drive tests, add the values shown below to your `.env` file with the values pulled from vault as indicated.
 
 ```
-AIRFLOW_VAR_GOOGLE_CONNECTION="google_cloud_default"
-AIRFLOW_VAR_GOOGLE_SERVICE_ACCOUNT_JSON=${get from vault at `rialto-airflow/prod/google_service_account_json`}
-AIRFLOW_CONN_GOOGLE_CLOUD_DEFAULT="google-cloud-platform://?keyfile_dict=${AIRFLOW_VAR_GOOGLE_SERVICE_ACCOUNT_JSON}&scope=https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fdrive%2Chttps%3A%2F%2Fwww.googleapis.com%2Fauth%2Fspreadsheets&project=sul-rialto&num_retries=5"
-AIRFLOW_VAR_GOOGLE_DRIVE_ID=${get from vault at `puppet/application/rialto-airflow/prod/google_drive_id`}
```

Comment:
these should already be in your .env file from the vault script in the steps above